templates/repo/branch/list.tmpl

```diff
@@ -44,9 +45,10 @@
 									{{else}}
 										<a href="{{$.RepoLink}}/src/branch/{{.Name | EscapePound}}">{{.Name}}</a>
 										<p class="time">{{$.i18n.Tr "org.repo_updated"}} {{TimeSince .Commit.Committer.When $.i18n.Lang}}</p>
-									</td>
 									{{end}}
+									</td>
 									<td class="ui">
+										{{if not .IsDeleted}}
```

Comment:
Is it necessary to render this empty `<td class="ui">` when this condition is false?

Reply:
Yes, as the list may contain existing and deleted branches.